code/SecurityAdmin.php

```diff
@@ -2,316 +2,66 @@
 
 namespace SilverStripe\Admin;
 
-use SilverStripe\Control\HTTPRequest;
-use SilverStripe\Control\HTTPResponse;
-use SilverStripe\Core\Convert;
+use Exception;
 use SilverStripe\Core\Injector\Injector;
-use SilverStripe\Forms\FieldList;
-use SilverStripe\Forms\Form;
 use SilverStripe\Forms\GridField\GridField;
-use SilverStripe\Forms\GridField\GridFieldConfig_RecordEditor;
-use SilverStripe\Forms\GridField\GridFieldDataColumns;
-use SilverStripe\Forms\GridField\GridFieldDetailForm;
-use SilverStripe\Forms\GridField\GridFieldExportButton;
 use SilverStripe\Forms\GridField\GridFieldImportButton;
-use SilverStripe\Forms\GridField\GridFieldPageCount;
-use SilverStripe\Forms\HiddenField;
-use SilverStripe\Forms\LiteralField;
-use SilverStripe\Forms\Tab;
-use SilverStripe\Forms\TabSet;
 use SilverStripe\Security\Group;
 use SilverStripe\Security\Member;
 use SilverStripe\Security\Permission;
 use SilverStripe\Security\PermissionProvider;
 use SilverStripe\Security\PermissionRole;
-use SilverStripe\View\ArrayData;
-use SilverStripe\View\Requirements;
 
 /**
  * Security section of the CMS
  */
-class SecurityAdmin extends LeftAndMain implements PermissionProvider
+class SecurityAdmin extends ModelAdmin implements PermissionProvider
 {
+    private static $managed_models = [
+        Member::class,
+        Group::class,
+        PermissionRole::class
+    ];
 
     private static $url_segment = 'security';
 
-    private static $url_rule = '/$Action/$ID/$OtherID';
-
     private static $menu_title = 'Security';
```

Comment:
```suggestion
    private static $menu_title = 'Security';

    private static $menu_priority = 0;
```
The menu priority is overridden in `ModelAdmin` to -0.5 which lumps all model admins together below the built-ins. We should keep it at 0 so it stays where it has always been.